tests/helpers/toLowerCase.test.js

```diff
@@ -1,11 +0,0 @@
-var helpers = require('../../js/main.js');
```

Comment:
There's 20 passing and 25 pending if you're feeling in the mood to write some tests :P ! .. Just run with `npm test`